cmd/twins/main.go

```diff
@@ -232,16 +248,32 @@ func connectToAuth(cfg config, logger logger.Logger) *grpc.ClientConn {
 	return conn
 }
 
-func newService(ps messaging.PubSub, chanID string, users mainflux.AuthNServiceClient, dbTracer opentracing.Tracer, db *mongo.Database, logger logger.Logger) twins.Service {
+func connectToRedis(cacheURL, cachePass, cacheDB string, logger logger.Logger) *redis.Client {
+	db, err := strconv.Atoi(cacheDB)
+	if err != nil {
```

Comment:
why dont  you return an error and handle it in the caller and thus no need for logger here?

Reply:
This is a copy-pasted func that we use elsewhere also. We use the same pass logger signature also in `connectToAuth`. Maybe it's better to change it everywhere. @dusanb94 ?

Reply:
The purpose of these functions inside `main.go` is to simplify the main method itself, so I would vote to keep it like it is.